notebooks/00_quick_start/sequential_recsys_amazondataset.ipynb

```diff
@@ -71,8 +71,10 @@
     "#from reco_utils.recommender.deeprec.models.sequential.asvd import A2SVDModel\n",
     "#from reco_utils.recommender.deeprec.models.sequential.caser import CaserModel\n",
     "#from reco_utils.recommender.deeprec.models.sequential.gru4rec import GRU4RecModel\n",
+    "#from reco_utils.recommender.deeprec.models.sequential.nextitnet import NextItNetModel\n",
     "\n",
     "from reco_utils.recommender.deeprec.io.sequential_iterator import SequentialIterator\n",
+    "#from reco_utils.recommender.deeprec.io.nextitnet_iterator import NextItNetIterator\n",
```

Comment:
Have you tested that when uncomment these lines and select NextItNet, the notebook will run normally with NextItNet? 
Meanwhile, can you add the running results to the table at the end of this notebook?

Reply:
It seems that Aamazon dataset does not have strong sequence, which is important for NextItNet